components/Layout/Images/__tests__/BlurhashCanvas.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
You can remove this and add the descriptions that were alongside ``@ts-expect-error`` comments so ESLint doesn''t complain